api4/command.go

```diff
@@ -156,8 +156,8 @@ func listCommands(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	commands := []*model.Command{}
-	err := &model.AppError{}
```

Comment:
Not erroneous, but generally non-nil error variables shouldn't be ignored. So raises some flags.